content/actions/creating-actions/metadata-syntax-for-github-actions.md

```diff
@@ -22,19 +22,19 @@ Docker and JavaScript actions require a metadata file. The metadata filename mus
 
 Action metadata files use YAML syntax. If you're new to YAML, you can read "[Learn YAML in five minutes](https://www.codeproject.com/Articles/1214409/Learn-YAML-in-five-minutes)."
 
-### **`name`**
+### `name`
```

Comment:
This actually does come out a little lighter, but I didn't dig into the CSS on why

Reply:
You're right, there's a slight visual difference, but that seems fine 🙂